docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,143 @@
+
+## Where to put the dagster_cloud.yaml file
+
+The dagster_cloud.yaml file should be placed in your dagster project's Github repository root. Below is an example of a file structure from the [Dagster cloud ETL quickstart](https://github.com/dagster-io/dagster/tree/master/examples/quickstart_etl).
+
+```shell
+quickstart_etl
+├── README.md
+├── quickstart_etl
+│   ├── __init__.py
+│   ├── assets
+├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+## Code Locations
+
+The only **mandatory** section in the dagster_cloud.yaml file is the locations key that specifies a list of code locations that should be deployed to Dagster Cloud. Each item in the list of locations must also include a location_name key that specifies the name of a location and a code_source key that specifies where to find the definitions in the code location.
+Parameter reference table:
+
+| Parameter                            | Description                                                                           | Data type            |
+| ------------------------------------ | ------------------------------------------------------------------------------------- | -------------------- |
+| `locations.location_name`            | The name of your code location that will appear in the Dagster UI code locations page | string               |
+| `locations.code_source.package_name` | Use if your dagster code is structured as a package                                   | string (folder name) |
+| `locations.code_source.python_file`  | Use if your dagster code is in a single file                                          | string (.py file)    |
+| `locations.code_source.module_name`  | The module that contains your dagster code (which matches pyproject.toml)                                            | string (module name) |
+
+Example for a single code location:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+```
+
+Example for multiple code locations:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+  - location_name: machine_learning
+    code_source:
+      python_file: ml/ml_model.py
+```
+
+## Configuring your working directory
+
+There might be some cases where you want to run your Dagster code from a different directory than the root of your code repository. You can specify the directory you want to run your Dagster code from in the dagster_cloud.yaml file using the `working_directory` option.
+
+Parameter reference table:
+
+| Parameter           | Description                                            | Data type     |
+| ------------------- | ------------------------------------------------------ | ------------- |
+| `working_directory` | The path of the directory that Dagster should run from | string (path) |
```

Comment:
I'm not gonna suggest a description re-write for this until we know what it does, but just marking this so we come back to it after it's been tested.